features/data/305.Big_Data_Spark/02.excludes

```diff
@@ -0,0 +1,9 @@
+root[0]['id']
+root[0]['notification']['timesSent']
```

Comment:
I think that root[0]['notification']['timesSent'] should be greatest or equal to 1

Reply:
Agree.

Reply:
The comment is because it should not be part of the excludes but check in the code to be ge 1

Reply:
The "excludes" are there in order not to be compared with a bigger json. The elements are not removed and they are compared, as expected, to be greater than 1.

Reply:
NTC